lectures/long_run_growth.md

```diff
@@ -230,24 +239,22 @@ def draw_interp_plots(series, ylabel, xlabel, color_mapping, code_to_name, lw, l
 
         # Plot the non-interpolated data with solid lines
         ax.plot(series[c],
-                linestyle='-',
                 lw=lw,
                 color=color_mapping[c],
                 alpha=0.8,
                 label=code_to_name.loc[c]['country'])
         
-        if logscale == True:
+        if logscale:
             ax.set_yscale('log')
     
     # Draw the legend outside the plot
-    ax.legend(loc='center left', bbox_to_anchor=(1, 0.5), frameon=False)
+    ax.legend(loc='upper left', frameon=False)
     ax.set_ylabel(ylabel)
     ax.set_xlabel(xlabel)
     
     return ax
```

Comment:
I think we can drop `return ax` here.  

The code below should then be modified accordingly.